### ISSUE TITLE
Waterfox' portable app opening with wrong user's Profile

### ISSUE THREAD

**Issue by gloschtla** [BrowserWorks/waterfox#1177]
I installed WaterfoxPortable_56.2.14_English.paf.exe into my local directory of my PortableApps collection. Then I used to open WaterFox via the start menue of PortableApps.com, then I customized it by adding addons and changing browser settings. Later I decided to set WaterFox as default browser. As expected, hyperlinks are consequently opened in waterfox portable. But whenever waterfox is opened via hyperlink, it opens without any of my previously customized settings. So there seem to be two user profiles now: one profile for opening waterfox via portableapps' start menue, and another profile for opening waterfox as default browser by clicking on hyperlinks. 

Wouldn't it be useful to have only one single user profile, independently of how WaterFox is started? 

This same issue also appeared in FirefoxPortable. It might either originate from mozilla's architecture, of from PortableApps'.

**Comment by grahamperrin** [comment]
Maybe related, although I know very little about portable applications: #1154

**Comment by MrAlex94** [comment]
The portable launcher directs Waterfox to the portable profile. Unfortunately Windows doesn't support symlinks, as you could have directed the profile the browser uses locally to the portable one. The profile manager doesn't support custom locations either to set the portable location as default.

You could try manually editing [profiles.ini](http://kb.mozillazine.org/Profiles.ini_file)? Edit the path to the location of the portable directory, and make sure  `IsRelative=1` for that profile.

**Comment by grahamperrin** [comment]
> `IsRelative=1`

Sanity checking, `1` or `0`? (Is the intention here to specify an absolute path?)

**Comment by MrAlex94** [comment]
> > `IsRelative=1`
> 
> Sanity checking, `1` or `0`? (Is the intention here to specify an absolute path?)

`1`, and it can only be used for a single profile. And yes.

**Comment by grahamperrin** [comment]
Checking my own sanity … I imagined that `1` implies `true`

**Comment by gloschtla** [comment]
I can add some more info: After having opened default browser Waterfox Portable via click on a hyperlink, I sticked its link to the taskbar. Whenever I start Waterfox through the taskbar, the problem remains. 

I compared the two instances of Waterfox (the first opened via portableapps menue and the other opened via taskbar) - so I checked about:profiles and about:addons 

The content of about:profiles is exactly the same, only about:addons is suddenly empty when having opened Waterfox via taskbar. Also all other customizing settings, I made through Waterfox opened via PortableApps menue, are gone.
